metricbeat/module/prometheus/query/query.go

```diff
@@ -83,9 +82,9 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 		m.http.SetURI(url)
 		response, err := m.http.FetchResponse()
 		if err != nil {
-			msg := fmt.Sprintf("unable to fetch data from prometheus endpoint: %v", pathConfig.Path)
-			m.Logger().Debugf("%v: %v", msg, err)
-			reporter.Error(errors.Wrap(err, msg))
+			err = errors.Wrapf(err, "unable to fetch data from prometheus endpoint: %v", pathConfig.Path)
+			m.Logger().Debug(err)
```

Comment:
Oh, sorry I forgot that this is not necessary anymore, you can simply report `error` and the reporter does the logging